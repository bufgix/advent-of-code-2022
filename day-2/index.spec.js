import { describe, expect, test } from "vitest";
import { solution1, solution2 } from "./index";
import { input } from "./input";

const testInput = `A Y
B X
C Z`;

describe("day 2", () => {
  test("q1", () => {
    expect(solution1(testInput)).toBe(15);
    expect(solution1(input)).toMatchSnapshot();
  });

  test("q2", () => {
    expect(solution2(testInput)).toBe(12);
    expect(solution2(input)).toMatchSnapshot();
  });
});
