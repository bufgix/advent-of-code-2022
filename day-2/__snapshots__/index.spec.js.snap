// Vitest Snapshot v1

exports[`day 2 > q1 1`] = `11767`;

exports[`day 2 > q2 1`] = `13886`;
